app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The association should be `belongs_to :author, class_name: 'User'`

Reply:
* done 👍